apps/next/app/dashboard/settings/[[...slug]]/page.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import { OrganizationProfile } from "@clerk/nextjs";
+import { DollarSign, IceCreamCone } from "lucide-react";
+import BillingPage from "./_profile-pages/billing-page";
+
+const OrganizationProfilePage = () => (
+  <OrganizationProfile path="/dashboard/settings" routing="path">
```

Comment:
only thing I don't like about this page is that I'd like it to be consistent with the clusters page and have a width that spans 2/3rds of the page. Could you try copying over the setup on that page and maybe playing with the `appearance` prop? https://clerk.com/docs/components/customization/overview#use-tailwind-classes-to-style-clerk-components

Reply:
Saw this after making the page full-width as seen below. Will work on making it 2/3 next week.
![Screenshot 2024-06-07 at 19 51 24](https://github.com/onmetal-dev/metal/assets/10248833/3fc31195-0907-45c9-90c0-076bfbd6fd3f)

Reply:
can you make a styles.css and import it into this page that hides the top left header ("Organization") and paragraph ("Manage your organization")? I don't see an option in the appearance prop to do this

```
h1[data-localization-key="organizationProfile.navbar.title"],
p[data-localization-key="organizationProfile.navbar.description"] {
  display: none;
}
```

Also can you add `headerTitle: "hidden"` to the appearance prop

Reply:
Done.